README.md

```diff
@@ -1,120 +1,31 @@
-# Map Colonies typescript service template
+# Job Manager Server
 
-----------------------------------
+This server provides a management interface for job trees and includes a RESTful API for interacting with a PostgreSQL database.
 
-This is a basic repo template for building new MapColonies web services in Typescript.
+**Key Features:**
 
-> [!IMPORTANT]
-> To regenerate the types on openapi change run the command `npm run generate:openapi-types`.
+* Manages job trees.
+* Provides a RESTful API for database management.
 
-## Development
-When in development you should use the command `npm run start:dev`. The main benefits are that it enables offline mode for the config package, and source map support for NodeJS errors.
+**Requirements:**
 
-### Template Features:
+* **PostgreSQL:** Version 13 or higher is required.
 
-- eslint configuration by [@map-colonies/eslint-config](https://github.com/MapColonies/eslint-config)
+------------------------
 
-- prettier configuration by [@map-colonies/prettier-config](https://github.com/MapColonies/prettier-config)
+## Migration
+**Development Workflow:**
 
-- jest
+* **Schema Validation:** `npm run migration:validate` - Ensures the integrity and correctness of your Prisma schema.
+* **Schema Formatting:** 
+    * `npm run migration:format` - Formats the Prisma schema for consistency and readability.
+    * `npm run migration:format-fix` - Automatically applies formatting changes to the schema.
+* **Migration Creation:** `npm run migration:dev` - Creates and applies a new migration to your local development database.
+* **Type Generation:** `npm run migration:generate` - Generates and updates TypeScript types for your database models.
 
-- .nvmrc
+**Production Deployment:**
+* **Migration Deployment:** `npm run migration:deploy` - Deploys the latest migrations to your production database.
```

Comment:
change to numbered list as its step by step

Reply:
done